sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -988,42 +909,26 @@ private void restorePseudonymizedDto(SampleDto dto, Sample existingSample, Sampl
 	}
 
 	private void pseudonymizeAssociatedObjects(
-		SampleJurisdictionDto sampleJurisdiction,
 		CaseReferenceDto sampleCase,
 		ContactReferenceDto sampleContact,
 		EventParticipantReferenceDto sampleEventParticipant,
-		Pseudonymizer pseudonymizer) {
+		Pseudonymizer pseudonymizer,
+		boolean isInJurisdiction) {
```

Comment:
The original code checks the jurisdiction of each associated object, now it does based on a single flag. I think this should be changed